src/test/java/core/basesyntax/service/DataReaderTest.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import core.basesyntax.service.impl.DataReaderCsvImpl;
+import java.util.List;
+import org.junit.Test;
+
+public class DataReaderTest {
+    private static final String DATA_PATH_OK = "src/test/resources/FruitShopOk.csv";
+    private static final String INCORRECT_PATH = "src/test/resssources/Fruit.csv";
+
+    @Test
+    public void readTest_Ok() {
+        List<String> expected = List.of(
+                "type,fruit,quantity",
+                "b,banana,20",
+                "b,apple,100",
+                "b,orange,150",
+                "s,banana,100",
+                "s,orange,20",
+                "p,banana,13");
+        DataReader dataReader = new DataReaderCsvImpl();
+        List<String> actual = dataReader.read(DATA_PATH_OK);
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void incorrectPathTest_NotOk() {
```

Comment:
Name your tests like this: methodUnderTest_state_expectedBehavior
```suggestion
    public void read_incorrectPath_notOk() {
```

Reply:
fix it everywhere